bolt-design-system/bolt
# Action Blocks | Testing | Update include path and add tests

## Jira

http://vjira2:8080/browse/BDS-1273

## Summary

Update the path to the `action-block` subcomponent. Add test to check that including the Twig subcomponent directly renders properly.

Add additional tests to check that deprecated props still work.

## Details

The `action-block.twig` adapter file was not updated when we changed the subcomponent filename from `_action-block.twig` to `_action-block-item.twig`. This update fixes that.

Also, the following props were deprecated and tests have been added to check that each one still works:
- `maxItemsPerRow` to `max_items_per_row`
- `align` to `valign`
- `border` to `borderless`

## How to test

- Review the files changed
- See the two new test snapshots and verify this is the HTML we would expect to see given the tests run
